renovate.json

```diff
@@ -1,5 +1,27 @@
 {
-  "extends": [
-    "config:base"
-  ]
+ "extends": ["config:base"],
+ "baseBranches": ["master"],
+ "bumpVersion": null,
+ "packageRules": [
+   {
+     "depTypeList": ["devDependencies"],
+     "updateTypes": ["minor"],
+     "automerge": false
+   }
+ ],
+ "prHourlyLimit": 5,
+ "prConcurrentLimit": 5,
+ "rangeStrategy": "bump",
+ "renovateFork": true,
+ "includeForks": true,
+ "regexManagers": [
+    {
+      "fileMatch": ["^lib/common.sh$"],
+      "matchStrings": ["MINIKUBE_VERSION:-\"(?<currentValue>.*?)\"}"],
+      "depNameTemplate": "kubernetes/minikube",
+      "datasourceTemplate": "github-releases"
+    }
+  ],
+  "assignees": ["kashifest", "fmuyassarov","furkatgofurov7", "Xenwar", "smoshiur1237", "jan-est"],
```

Comment:
We need to add everyone in the team may be?

Reply:
What I've done here is the intersection of reviewers / approvers in the OWNERS file with our team. I could add all the members of our team regardless of whether they have permissions in this repo to approve or lgtm the changes?

Reply:
Make sense, but irrespective of approve/lgtm rights, reviewer can anyhow review the code was point

Reply:
@macaptain I think you can add `reviewers` option to add team members who don't have right to approve or lgtm. Using this approach allows adding a person who has the right, and a person who may not have the privilege. 
https://docs.renovatebot.com/configuration-options/#reviewers

Reply:
@namnx228 that looks like a good solution. Since I have to add more dependencies to Renovate, I will add `reviewers` in the next update.